RocketChat/Rocket.Chat.ReactNative
# fix: remove blur from attachments

## Proposed changes

As mentioned here: https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/5420#issuecomment-1864869943
The react-native-blur component has a performance impact on Android devices. Therefore, to improve performance, we opted to replace the blur view with a simple view set to 80% opacity for the background, as per the design team's recommendation.

## Issue(s)	

Closes https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/5420

## How to test or reproduce


- Go to a direct chat with somebody where a lot of images were shared.
- Just use the app a bit: go back; observe your chats

## Screenshots

#### Before
![image](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/bbf8f38e-f33e-4af0-bfae-63f4cc793316)

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/a14c7971-23d9-4090-b4a8-ee4e809fb9d6

#### After
![image](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/221d11e2-bcec-4410-a38d-337ce0ce8012)

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/04a1e594-92ea-4980-9fcc-e8280b84637c

## Types of changes




## Checklist



## Further comments

[TC-1066]

[TC-1066]: https://rocketchat.atlassian.net/browse/TC-1066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
